app/.prettierrc.json

```diff
@@ -1,4 +1,5 @@
 {
   "semi": false,
-  "singleQuote": true
+  "singleQuote": true,
+  "trailingComma": "none"
```

Comment:
Are these here because ESLint complains? I don't feel super strongly, but have a slight preference for an opposite approach: disabling style lint rules and using whatever the code formatter's default is. We achieved this on PFML by adding https://github.com/prettier/eslint-config-prettier – kinda outta scope for this i18n PR though.

Reply:
Sounds good. I'll switch the approach